CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj

```diff
@@ -246,7 +246,4 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="..\CefSharp.props" />
   <Import Project="..\CefSharp.Native.props" />
-  <PropertyGroup>
```

Comment:
I believe this is necessary in order to run the WPF example project from within the Mailbird solution. Try cleaning your solution and see if you can get it running without this (to confirm this isn't necessary).

Reply:
Looks like they moved these resources online, for example https://cefsharp.example/BindingTest.html